File: Conditionals/OldEnough.py
def check_eligibility(age):
    if age >= 18:
        print("You are old enough to vote, drive, and get a learner's permit.")
    elif age >= 16:
        print("You are old enough to drive and get a learner's permit.")
    elif age >= 15:
        print("You are old enough to get a learner's permit.")
    elif age >= 5:
        print("You are old enough to go to school.")
    else:
        print("You are not old enough for any of the conditions.")

user_age = int(input("Enter your age: "))
check_eligibility(user_age)

File: Conditionals/OldEnough.c
#include <stdio.h>

void checkEligibility(int age) {
    if (age >= 18) {
        printf("You are old enough to vote, drive, and get a learner's permit.\n");
    } else if (age >= 16) {
        printf("You are old enough to drive and get a learner's permit.\n");
    } else if (age >= 15) {
        printf("You are old enough to get a learner's permit.\n");
    } else if (age >= 5) {
        printf("You are old enough to go to school.\n");
    } else {
        printf("You are not old enough for any of the conditions.\n");
    }
}

int main() {
    int user_age;
    printf("Enter your age: ");
    scanf("%d", &user_age);
    
    checkEligibility(user_age);
    
    return 0;
}